src/Network.js

```diff
@@ -195,7 +195,7 @@ class Network {
       // loop over the training data summing the error of all samples
       // http://www.researchgate.net/post
       //   /Neural_networks_and_mean-square_errors#rgw51_55cb2f1399589
-      this.error = _.sum(_.map(data, sample => {
+      this.error = _.sum(data, sample => {
```

Comment:
`_.sum` takes an array, `data` was already an array.  There is no need to `_.map` it first.